CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
-# Contributing to MeasurementLink™ Support for LabVIEW
+# Contributing to Measurement Plug-In Support for LabVIEW
```

Comment:
@rfriedma 
My understanding is we'll call the 'title' of the repo `Measurement Plug-In SDK for LabVIEW` as a whole. Ryan can you confirm this? Hopefully is just a global replace for this whole phrase.

Reply:
Confirmed with Ryan and updated it.